package.json

```diff
@@ -27,6 +27,7 @@
     "dids": "^2.4.0",
     "express": "^4.17.1",
     "forever": "^4.0.1",
+    "glmw": "^0.1.3",
```

Comment:
Let's merge the WASM functionality we need into our own wasm build; we should keep the number of WASM modules to a minimum.